bin/p2-schedule/main.go

```diff
@@ -53,12 +53,14 @@ func main() {
 			log.Fatalf("Could not read manifest at %s: %s\n", manifestPath, err)
 		}
 		path := kp.IntentPath(*nodeName, manifest.ID())
-		if *hookTypeName != "" {
+		if *hookTypeName == "global" {
```

Comment:
How about renaming the flag to `--hook`. If the value is the empty string, then we know the schedule command was `p2-schedule --hook`, and know to schedule it as a global hook. This way there doesn't need to be a sentinel value for global hooks, and will make migrating to an all-global hook world easier